src/main/java/com/mewna/catnip/shard/manager/DefaultShardManager.java

```diff
@@ -46,21 +45,18 @@
 import java.util.*;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
-import static com.mewna.catnip.shard.ShardAddress.CONTROL;
-import static com.mewna.catnip.shard.ShardAddress.computeAddress;
-import static com.mewna.catnip.shard.ShardControlMessage.CONNECT;
-
 /**
  * @author amy
  * @since 8/15/18.
  */
 @Accessors(fluent = true)
 public class DefaultShardManager extends AbstractShardManager {
     private final Collection<MessageConsumer> consumers = new HashSet<>();
-    private final Map<Integer, String> shards = new ConcurrentHashMap<>();
+    private final Map<Integer, CatnipShardImpl> shards = new ConcurrentHashMap<>();
```

Comment:
The diff for CatnipShard/Impl is kinda hard to read; why is this storing an Impl instead of the interface?

Reply:
start/stop methods

Reply:
:ok_hand: